src/safeposix/syscalls/net_calls.rs

```diff
@@ -603,7 +605,7 @@ impl Cage {
                             if let None = sockfdobj.localaddr {
                                 let bindret = sockobj.bind(&ladr);
                                 if bindret < 0 {
-                                    let sockerrno = match Errno::from_discriminant(unsafe{*libc::__errno_location()} as i32) {
+                                    match Errno::from_discriminant(unsafe{*libc::__errno_location()} as i32) {
```

Comment:
This seems to do some stuff that we want via the interface.

Reply:
Agreed, since we want to restrict unsafe code to the interface.